modin/pandas/series.py

```diff
@@ -484,7 +484,12 @@ def values(self):  # noqa: RT01, D200
         """
         Return Series as ndarray or ndarray-like depending on the dtype.
         """
-        return self.to_numpy()
+        import modin.pandas as pd
+
+        data = self.to_numpy()
+        if isinstance(self.dtype, pd.CategoricalDtype):
+            data = pd.Categorical(data, dtype=self.dtype)
+        return data
```

Comment:
this is still going to mess up a bunch of EA cases.  The correct way to handle this is https://github.com/modin-project/modin/issues/4187#issuecomment-1284391798

Reply:
@jbrockmendel Is there some simple condition by which we can determine what this is EA case?

Reply:
`isinstance(self.dtype, ExtensionDtype)`

Reply:
thanks @jbrockmendel! Should be fixed in https://github.com/modin-project/modin/pull/5493.